tools/mo/openvino/tools/mo/convert_impl.py

```diff
@@ -405,6 +399,11 @@ def prepare_ir(argv: argparse.Namespace):
     if moc_front_end:
         fallback_reasons = check_fallback(argv)
         if len(fallback_reasons) == 0:
```

Comment:
@rkazants I suggest to move creating of tmp .pb file here, because otherwise if we fallback to legacy path (due to usage of legacy extensions for example), .pb file is not removed. 
tmp .pb file is not used for legacy path, so I suggest to create it after we checked that fallback not needed.